### PR TITLE
SECURITY.md: make the process more concrete

### PR DESCRIPTION
**Pull request by AkihiroSuda**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -2,5 +2,11 @@
...
-If you do not have a GitHub account, you may use email to reach out to the [maintainers](https://lima-vm.io/docs/community/governance/) directly.
-(See the git commit logs to inspect the maintainers' email address.)
+If you do not have a GitHub account, you may use email to reach out to the [Committers](https://lima-vm.io/docs/community/governance/) directly.
+See the GitHub profiles or the git commit logs to inspect the Committers' email addresses.
+
+## Handling security issues
+The [Committers](https://lima-vm.io/docs/community/governance/) will try to triage the report and make the first response within 7 days.
+Releasing a fix may take a longer time, and may need further assistance from the reporter.
+
+If you do not see any response after 7 days, ping the Commiters via email or [Slack](https://lima-vm.io/docs/community/) privately.
```
Comment (jandubois) [inline SECURITY.md]:
```suggestion
If you do not see any response after 7 days, ping the Committers via email or [Slack](https://lima-vm.io/docs/community/) privately.
```

**Review by jandubois — CHANGES_REQUESTED** [review]
There is a typo.

It is also not clear how the reporter can identify committers on Slack. I think we mostly have similar Slack and GitHub usernames, but that cannot be guaranteed. Do we need to list our CNCF Slack usernames explicitly in the maintainers document?

**Comment by jandubois** [comment]
> Do we need to list our CNCF Slack usernames explicitly in the maintainers document?

Oops, I just realized that we can be found by our regular names, not handles, so this should be good enough.

File: SECURITY.md (hunk 1 above)
Comment (AkihiroSuda) [reply SECURITY.md]: fixed

**Review by jandubois — APPROVED** [review]
Thanks, LGTM

**Pull request merged**